docs/source/typing_anti_pitch.rst

```diff
@@ -0,0 +1,80 @@
+.. _typing-anti-pitch:
+
+Reasons to avoid static type checking
+=====================================
+
+In the words of :pep:`484`::
+
+    It should also be emphasized that Python will remain a dynamically typed language, and the
+    authors have no desire to ever make type hints mandatory, even by convention.
+
+The idea that dynamism in Python is a strength of the language is reflected in the fact that
+Python's type system is gradual. See :pep:`483` for details, but the long and short of this is
+that you can add static types to your codebase only to the extent that you want to, and static
+type checkers are designed to be able to put up with this.
```

Comment:
It’s not just static type checkers, it’s also IDEs, and maybe other tools as typing gains popularity.

And to cover those other tools, “are designed” should probably be changed to “should be designed”. That would allow directing complaints like “my IDE needs type hints to do autocompletion” towards the IDE to advise the user.